### PR TITLE
all eris-clis turned into eris

### PR DESCRIPTION
**Pull request by VoR0220**
* fixes naming of the imports...bludgeoned it a bit...let me pick through and see what needs fixing.

### CODE REVIEW

File: chains/chains.go [hunk 1]
```diff
@@ -173,7 +173,7 @@ func CatChain(do *definitions.Do) error {
 		doCat.Operations.Args = []string{"cat", path.Join(rootDir, "genesis.json")}
 	case "config":
 		doCat.Operations.Args = []string{"cat", path.Join(rootDir, "config.toml")}
-	// TODO re-implement with eris-client ... mintinfo was remove from container (and write tests for these cmds)
+	// TODO re-implement with erisent ... mintinfo was remove from container (and write tests for these cmds)
```
Comment (VoR0220) [inline chains/chains.go]: put this back to eris-client

File: cmd/chains.go [hunk 2]
```diff
@@ -272,7 +272,7 @@ var chainsCat = &cobra.Command{
 	Example: `$ eris chains cat simplechain config -- display the config.toml file from inside the container
 $ eris chains cat simplechain genesis -- display the genesis.json file from the container`,
 	// [zr] these don't work (mintinfo not found in container)
-	// TODO re-implement when eris-client is merged into edb
+	// TODO re-implement when erisent is merged into edb
```
Comment (VoR0220) [inline cmd/chains.go]: turn back to eris-client

File: misc/release/Dockerfile-rpm [hunk 3]
```diff
@@ -33,7 +33,7 @@ WORKDIR /root
 
 COPY linux-private-key.asc /root
 COPY linux-public-key.asc /root
-COPY eris-cli.spec /root/rpmbuild/SPECS/
+COPY eris.spec /root/rpmbuild/SPECS/
```
Comment (zramsay) [inline misc/release/Dockerfile-rpm]: has the actual filename been changed?

**Comment by VoR0220** [comment]
Caaaaaaaaaaan do

**Comment by VoR0220** [comment]
done

**Pull request merged**